src/MigrationTools.Host/StartupService.cs

```diff
@@ -16,14 +17,16 @@ public interface IStartupService
 
     internal class StartupService : IStartupService
     {
+        private readonly IHostApplicationLifetime _LifeTime;
         private readonly IDetectOnlineService _detectOnlineService;
         private readonly IDetectVersionService _detectVersionService;
         private readonly ILogger<StartupService> _logger;
         private readonly ITelemetryLogger _telemetryLogger;
         private static Stopwatch _mainTimer = new Stopwatch();
 
-        public StartupService(IDetectOnlineService detectOnlineService, IDetectVersionService detectVersionService, ILogger<StartupService> logger, ITelemetryLogger telemetryLogger)
+        public StartupService(IHostApplicationLifetime lifeTime, IDetectOnlineService detectOnlineService, IDetectVersionService detectVersionService, ILogger<StartupService> logger, ITelemetryLogger telemetryLogger)
         {
+            _LifeTime = lifeTime;
```

Comment:
pascal casing for variables, do we need to align this?

Reply:
Is it not camel casing when the underscore is lowercase?

Reply:
the underscore is not counted ;) but all the other variables here starts with lowercase so good to be in sync in the same file. But I have seen both ways used in the repo. Which type are we selecting going forward?
1. _MyVariable
2. _myVariable

Reply:
My visual studio created #1 for me

Reply:
But do you like the number 1 or number 2? I think we can create a .editorconfig that creates number1 for you.